etc/dnf/aliases.d/command-abbreviations.conf

```diff
@@ -0,0 +1,19 @@
+[aliases]
+dup = distro-sync
+dsync = distro-sync
+dist-upgrade = distro-sync
+dg = downgrade
+in = install
+mc = makecache
+ref = makecache
+refresh = makecache
+rei = reinstall
+ri = reinstall
+rm = remove
+se = search
+sh = shell
+up = upgrade
+u-m = upgrade-minimal
+up-min = upgrade-minimal
+um = upgrade-minimal
```

Comment:
Same like for distro-sync

Reply:
Let's just keep `up-min` for `upgrade-minimal`.

Reply:
@pkratoch hey Pavla, would you please consider the change proposed by @Conan-Kudo above?

Reply:
Sorry, done.